google/cloud/compute/v1small/BUILD.bazel

```diff
@@ -0,0 +1,55 @@
+# For the converter's Bazel rules testing and demonstration, specifically
+# this file shows to to use the `proto_from_disco` build rule defined in this
+# repository.
+load(
+    "//rules_gapic:disco_to_proto.bzl",
+    "gapic_yaml_from_disco",
+    "grpc_service_config_from_disco",
+    "proto_from_disco",
+)
+
+_MESSAGE_IGNORE_LIST = [
+    "HttpHealthCheck",
+    "HttpsHealthCheck",
+    "HttpHealthCheckList",
+    "HttpsHealthCheckList",
+    "GetHttpHealthCheckRequest",
+    "GetHttpsHealthCheckRequest",
+    "PatchHttpHealthCheckRequest",
+    "PatchHttpsHealthCheckRequest",
+    "UpdateHttpHealthCheckRequest",
+    "UpdateHttpsHealthCheckRequest",
+    "InsertHttpHealthCheckRequest",
+    "InsertHttpsHealthCheckRequest",
+    "ListHttpHealthChecksRequest",
+    "ListHttpsHealthChecksRequest",
+    "DeleteHttpHealthCheckRequest",
+    "DeleteHttpsHealthCheckRequest",
+]
+
+_SERVICE_IGNORE_LIST = [
+    "HttpHealthChecks",
+    "HttpsHealthChecks",
+]
+
+proto_from_disco(
+    name = "compute",
+    src = "//:src/test/resources/compute.v1.json",
```

Comment:
Are your files swapped? This file is under `compute/v1small/` and references the full json, while the other one iunder`/v1/` refers to to `compute.v1small.json`

Reply:
oh, yeah, thanks! those files exist to simply run the generator without asserts (just a sanity check that the rules at least do not fail themselves. That is why even swapped version is "ok" but yes, it definitely should be cleaned up